docs/api/doc-block-source.md

```diff
@@ -102,7 +102,16 @@ Type: `boolean | 'dedent' | BuiltInParserName`
 
 Default: `parameters.docs.source.format` or `true`
 
-Specifies the formatting used on source code. Supports all valid [prettier parser names](https://prettier.io/docs/en/configuration.html#setting-the-parserdocsenoptionshtmlparser-option).
+Specifies the formatting used on source code. Both `true` and `'dedent'` have the same effect of removing any extraonous indentation. Supports all valid [prettier parser names](https://prettier.io/docs/en/configuration.html#setting-the-parserdocsenoptionshtmlparser-option).
+
+### `transform`
```

Comment:
This prop should be in alphabetical order with the others, for consistency.

Reply:
Also "extraneous"